documentation/migration-guides/Az.5.0.0-migration-guide.md

```diff
@@ -182,38 +205,50 @@ New-AzFunctionApp -ResourceGroupName $rd `
 ## Az.KeyVault
 
 ### New-AzKeyVault
+
 No longer supports the parameter `DisableSoftDelete` and no alias was found for the original parameter name.
 
 #### Before
+
 ```powershell
 # Opt out soft delete while creating a key vault
 New-AzKeyVault -VaultName 'Contoso03Vault' -ResourceGroupName 'Group14' -Location 'East US' -DisableSoftDelete
 ```
 
 #### After
-The ability to update soft-delete setting is deprecated in Az.KeyVault 3.0.0. Read more: https://docs.microsoft.com/en-us/azure/key-vault/general/soft-delete-change
+
+The ability to update soft-delete setting is deprecated in Az.KeyVault 3.0.0. Read more: https://docs.microsoft.com/azure/key-vault/general/soft-delete-change
 
 ### Update-AzKeyVault
-No longer supports the parameter `EnableSoftDelete`, `SoftDeleteRetentionInDays` and no alias was found for the original parameter name.
+
+No longer supports the parameter `EnableSoftDelete`, `SoftDeleteRetentionInDays`, and no alias was
+found for the original parameter name.
 
 #### Before
+
 ```powershell
 Update-AzKeyVault -VaultName 'Contoso03Vault' -ResourceGroupName 'Group14' -EnableSoftDelete -SoftDeleteRetentionInDays 15
 ```
 
 #### After
-The ability to update soft-delete setting is deprecated in Az.KeyVault 3.0.0. Read more: https://docs.microsoft.com/en-us/azure/key-vault/general/soft-delete-change
+
+The ability to update soft-delete setting is deprecated in Az.KeyVault 3.0.0. Read more: https://docs.microsoft.com/azure/key-vault/general/soft-delete-change
 
 ### Get-AzKeyVaultSecret
-The property 'SecretValueText' of type 'Microsoft.Azure.Commands.KeyVault.Models.PSKeyVaultSecret' has been removed.The property 'SecretValueText' of type 'Microsoft.Azure.Commands.KeyVault.Models.PSDeletedKeyVaultSecret' has been removed.
+
+The property `SecretValueText` of type `Microsoft.Azure.Commands.KeyVault.Models.PSKeyVaultSecret`
+has been removed. The property `SecretValueText` of type
+`Microsoft.Azure.Commands.KeyVault.Models.PSDeletedKeyVaultSecret` has been removed.
```

Comment:
Same sentence has been repeated twice and we should explain that `SecretValueText` has been replaced with `SecretValue`.